lib/form-field.css

```diff
@@ -117,16 +115,17 @@
   display: flex;
 }
 
-.FormField-check .FormField-input {
-  margin-right: var(--FormField-check-gutter);
+.FormField-checkInput {
+  box-sizing: border-box;
+  font-size: inherit;
+  margin: 0 var(--FormField-check-gutter) 0 0;
```

Comment:
Would we not want to share some of the base styles from [line 48](https://github.com/simonsmith/suitcss-components-form-field/pull/1/files#diff-dc4bb943adf689169ccc157002c24badR48)?

Reply:
These are all the necessary rules for radio/checkbox inputs far as I can tell.

Reply:
I was wondering more if it was worth grouping them, so:

```css
.FormField-select,
.FormField-input,
.FormField-checkInput {
  /* styles that are common */
}

.FormField-select,
.FormField-input {
  /* specific */
}

.FormField-checkInput {
  /* specific */
}
```

And I'm assuming we keep the padding as `0` because it's not useful to change for a check?

Reply:
Sure, pushed that change.

The `padding: 0` is to remove default padding [in IE 10](https://github.com/necolas/normalize.css/blob/master/normalize.css#L265) for consistent baseline. We could use `--FormField-input-padding` but that would give us a larger default style, this PR was just to remove all styles affecting the baseline.